COMPILING.md

```diff
@@ -114,6 +114,25 @@ files.
    complete set of built binaries can be found in `./build/bin` once the build
    is complete.
 
+## Compilation on Windows using Visual Studio
```

Comment:
:pick: I don't think this section should really be just about *compilation* with Visual Studio as much as its use as an IDE. As such I suggest that this new section be moved be moved just above the existing section on "WORKING WITH ECLIPSE" and that both sections be made subsections of a new "Working on CBMC using IDEs" section.

:bulb: We could also add a new "Working with CLion" section as a later enhancement (as a separate piece of work in a potential follow-up PR).